### ISSUE TITLE
[doc] Unclear what local refers to in installation docs

### ISSUE THREAD

**Issue by ahmetb** [kubernetes-sigs/kwok#333]
### What would you like to be added?

https://kwok.sigs.k8s.io/docs/user/kwok-in-local/ this doc doesn't make it clear what "in the local" means. For example, can it work on a Mac or Windows WSL where Docker is installed and running?  Or does it need a Kubernetes cluster?

e.g. If I run the command specified directly, it tries to talk to a local kind cluster (my current-context set on kubectl) but it fails because it's down

```
$ kwok --kubeconfig=~/.kube/config  ... # the command in the docs

  ERROR Failed to list nodes                                    err="Get \"https://127.0.0.1:45679/api/v1/nodes?limit=1\": dial tcp 127.0.0.1:45679: connect: connection refused"
  ERROR Failed to list nodes                                    err="Get \"https://127.0.0.1:45679/api/v1/nodes?limit=1\": dial tcp 127.0.0.1:45679: connect: connection refused"
  ...
```

### Why is this needed?

User experience.

This output makes me think this tool clearly needs a local Kubernetes cluster to be able to run, because it's trying to use the local Kubernetes cluster.

* But isn't the tool for bringing up a fake apiserver plus some components locally? Why does it need Kubernetes to do that (like `kind`)?
* And how is this different than ["Kwok in a Cluster"](https://kwok.sigs.k8s.io/docs/user/kwok-in-cluster/) if it needs a k8s cluster to run anyway?

Long story short, I think I should be using `kwokctl create cluster` command, but why is the "Kwok on local" page telling me to install it to my Mac via Homebrew and use `kwok` executable instead?

**Comment by wzshiming** [comment]
It makes sense, we will improve the documentation. 😄

**Comment by ahmetb** [comment]
I don't think #336 fixes this. Please read the comment above.

**Comment by wzshiming** [comment]
Ohh, sorry I didn't notice that PR was linked to this issue. 😓

/reopen

**Comment by windsonsea** [comment]
Hi @ahmetb , could you help to review #356? More comments can improve the doc, thanks.
